Automattic/pocket-casts-ios
# End of Year: Epilogue Grand Finale Idea

| 📘 Project: #376 | 🛫 Depends on: #566 |
|:---:|:---:|

This implements a bit more pizazz on the epilogue view based on the design review notes:
- Changes background to black
- Adds confetti
- Adds a shiny effect to the heart to add a pop of interactive color
- Adds haptics when the confetti appears

## Video

https://user-images.githubusercontent.com/793774/205430061-8b2dbb6f-cfcd-4e9e-ab4d-39ed6857e6cc.MP4


## To test

1. You should test this on a real device
2. Go to Profile > End of Year Card
3. Tap through to the second to last view
4. Let the last story play out, and hold your device in your hand
5. ✅ Verify the epilogue appears with some extra flare
7. ✅ Verify the haptics play and feel cool
6. ✅ Tilt your device around for extra effects

## Checklist
